src/pages/AdditionalInformation/PrivacyNotices.jsx

```diff
@@ -11,29 +11,26 @@ const TermsOfUse = () => {
         <div className='box-terms'>
           <div className='list-terms'>
             <NavLink
-              to='/termsofuse'
+              to='/faq'
               activeClassName='active'
               className='text-for-terms-use'
             >
               Često postavljana pitanja
             </NavLink>
-            <br />
             <NavLink
               to='/privacynotices'
               activeClassName='active'
               className='text-for-terms-use'
             >
               Obaveštenje o privatnosti
             </NavLink>
-            <br />
             <NavLink
```

Comment:
You need to align properly these `NavLink` components. See this in app
![Screenshot 2023-01-24 at 11 41 00](https://user-images.githubusercontent.com/85338076/214273791-707cb352-ab3b-4d7d-b69d-30693e134ba3.png)

Reply:
Changes are found on the fixed impressions component